examples/topic_models/lntm_mcem.py

```diff
@@ -115,6 +125,51 @@ def e_obj(observed):
     for i in params:
         print(i.name, i.get_shape())
 
+    # Below is the evaluation part.
+    # Variables whose name starts with '_' is only used in the evaluation part,
+    # to be distinguished from those variables used in the training part above.
+
+    _D = X_test.shape[0]
```

Comment:
We can assume testing batch size = training batch size right now.

Alternatively, we can make D as a placeholder.